src/Browscap/Data/Expander.php

```diff
@@ -43,6 +43,11 @@ class Expander
     private $logger = null;
 
     /**
+     * @var array
```

Comment:
data structures are hard :) let's leave it as is then, is it possible to comment on what the structure should look like at least?